client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -208,7 +209,11 @@ export const useStoreItemInfo = ( {
 				shoppingCartTracker( 'calypso_jetpack_shopping_cart_add_product', {
 					productSlug: item.productSlug,
 				} );
-				return addProductsToCart( [ { product_slug: item.productSlug } ] );
+				addProductsToCart( [ { product_slug: item.productSlug } ] );
+
+				const addedToCartText = translate( 'added to cart' );
```

Comment:
Maybe we should put it before `addProductsToCart` to still be able to return it?

Reply:
Since this is just an event handler, I felt there was not need to return anything, but I have addressed the suggestion in the new commit. Thanks